File: php/program/Dpr.php
<?php
/* Saya Nur Ainun Nim 2202046 mengerjakan soal Latihan 1 dalam mata kuliah Desain dan Pemrograman Berorientasi Objek 
untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin. */

class Dpr{
    //Private attributes 
    private $id = '';
    private $nama = '';
    private $bidang = '';
    private $partai = '';
    private $foto = '';

    //Public methods
    /*Constructors*/
    //Methods  set attributes dari class Dpr dengan parameter
    public function __construct($id = '', $nama = '', $bidang = '', $partai = '', $foto = '')
    {
        //Set attributes pada class Dpr sesuai isi parameter
        $this->id = $id;
        $this->nama = $nama;
        $this->bidang = $bidang;
        $this->partai = $partai;
        $this->foto = $foto;
    }

    //setter id 
    public function setId($id){
        $this->id = $id;
    }
    //getter id
    public function getId(){
        return $this->id;
    }

    //setter nama
    public function setNama($nama){
        $this->nama = $nama;
    }
    //geter nama
    public function getNama(){
        return $this->nama;
    }

    //setter bidang 
    public function setbidang($bidang){
        $this->bidang = $bidang;
    }
    //getter bidang
    public function getbidang(){
        return $this->bidang;
    }

    //setter partai 
    public function setpartai($partai){
        $this->partai = $partai;
    }
    //getter partai
    public function getpartai(){
        return $this->partai;
    }

    //setter  foto 
    public function setFoto($foto){
        $this->foto = $foto;
    }
    //getter foto
    public function getFoto(){
        return $this->foto;
    }
}

?>

File: java/program/Dpr.java
/*Saya Nur Ainun 2202046 mengerjakan soal Latihan 1 dalam mata kuliah Desain dan Pemrograman Berorientasi Objek 
untuk keberkahanNya maka saya tidak melakukan kecurangan seperti yang telah dispesifikasikan. Aamiin. 
*/

public class Dpr {
    // private attribute
    private String id;
    private String name;
    private String bidang;
    private String partai;

    //constructor. Take note that it doesn't have any return type.
    public Dpr()
    {
        this.id = "";
        this.name = "";
        this.bidang = "";
        this.partai = "";
    }

    // another constructor whit parameter.
    public Dpr(String id, String name, String bidang, String partai) 
    {
        this.id = id;
        this.name = name;
        this.bidang = bidang;
        this.partai = partai;
    }

    //Getter and Setter
    public String getId()
    {
        return this.id;
    }

    //set id
    public void setId(String id)
    {
        this.id = id;
    }

    // get name
    public String getName()
    {
        return this.name ;
    }

    // set name
    public void setName(String name)
    {
        this.name = name;
    }

    //get bidang
    public String getBidang()
    {
        return this.bidang;
    }

    public void setBidang(String bidang)
    {
        this.bidang = bidang;
    }

    //get partai
    public String getPartai()
    {
        return this.partai;
    }

    //set partai
    public void setPartai(String Partai)
    {
        this.partai = Partai;
    }
}